docs/contexts.md

```diff
@@ -875,18 +882,18 @@ With our new plugs in place, we can now modify our `create`, `update`, and `dele
     end
   end
 
-  def update(conn, %{"page" => page_params}) do
++ def update(conn, %{"page" => page_params}) do
```

Comment:
We should use `plus` signs to mark all changes that are need to be done in the CMS.PageController.

In this line, the `update` function is generated as `def update(conn, %{"id" => id, "page" => page_params})`. So we need to remove `id` and because of that the `plus` sign was added